vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/GraphQLSocketHandler.java

```diff
@@ -0,0 +1,16 @@
+package io.vertx.ext.web.handler.graphql;
+
+import graphql.GraphQL;
+import io.vertx.core.Handler;
+import io.vertx.core.http.ServerWebSocket;
+import io.vertx.ext.web.handler.graphql.impl.GraphQLSocketHandlerImpl;
+
+public interface GraphQLSocketHandler extends Handler<ServerWebSocket> {
+
+  GraphQLSocketHandler endHandler(Handler<ServerWebSocket> endHandler);
+
+  static GraphQLSocketHandler create(GraphQL graphQL) {
```

Comment:
Users may want to send queries and mutations over WS but no subscriptions. How would we handle this case? How about `ApolloWSHandlerOptions` with `subscriptionEnabled` field?

Reply:
Does it make sense have this option? Maybe just not defining suscriptions in the GraphQL specification would be enough.

Reply:
Fair enough.